ocaml/xapi/workload_balancing.ml

```diff
@@ -675,7 +675,7 @@ let get_evacuation_recoms ~__context ~uuid =
       []
     else
       match inner_xml with
-      | (Xml.Element ( _, _, [Xml.Element("CanPlaceAllVMs",_,_)])) ->
+      | (Xml.Element ( _, _, [_;_;Xml.Element("CanPlaceAllVMs",_,_)])) ->
```

Comment:
Please expand the commit message. It is not obvious how this change is related to the subject line. You gave an explanation in a comment - this should go into the commit message if it is essential to understand this change.